README.md

```diff
@@ -69,6 +69,7 @@ It can be installed using yay as shown below.
 ```
 yay -S lxqt-organizer
 ```
+
```

Comment:
Why?

Reply:
I don't see any reason for a newline. In case there is, @ito32bit please explain